File: Difficulty: Basic/Largest in Array/largest-in-array.c
// User function template for C
int largest(int arr[], int n) {
    // Code Here
    int max = arr[0];
    for(int i = 0; i <n ; i++) {
        if (arr[i] > max) {
            max = arr[i] ;
        }
    }
    return max;
}

File: Difficulty: Basic/Largest in Array/largest-in-array.java
class Solution {
    public static int largest(int[] nums) {
        // code here
        int n = nums.length;
        
        if (n==1) return nums[0];
        
        int maxi = nums[0];
        
        for(int i =0; i<n ; i++) {
            if(nums[i] > maxi) {
                maxi = nums[i];
            }
        }
        
        return maxi;
    }
}
